builtin/providers/aws/resource_aws_lambda_function.go

```diff
@@ -586,3 +587,14 @@ func validateRuntime(v interface{}, k string) (ws []string, errors []error) {
 	}
 	return
 }
+
+func validateTimeout(v interface{}, k string) (ws []string, errors []error) {
+	timeout := v.(int)
+
+	if timeout > 300 {
+		errors = append(errors, fmt.Errorf(
+			"Lambda timeout can't exceed 300. Currently set to %d.",
```

Comment:
ok sounds good to me - i love all the PRs you are sending so please do continue :)

We love it when community members help to make the product better!

Paul